### PR TITLE
adding steps for image mirroring

### PR DESCRIPTION
**Pull request by laurafitzgerald**
## JIRA
https://issues.redhat.com/browse/INTLY-4819

### CODE REVIEW

File: openshift-ci/README.md [hunk 1]
```diff
@@ -33,4 +33,27 @@ After making a change to the base image configuration in your repo, trigger a ne
...
+
+### Credentials
+
+To add your repo, update integreatly+prow robot to have read/write access to the repo.
+
+You shouldn’t need to add further credentials as the [integreatly+prow robot](https://quay.io/organization/integreatly?tab=robots)’s credentials are already sent to the DPTP team for [inclusion in ci](https://github.com/openshift/release/pull/6773/files).
+
+If for some reason you do need to add credentials for another account/bot, here are the steps:
+
+Ask @nmoraitis in the #forum-testplatform channel on slack for his public gpg key and tell him that you need credentials added.
```
Comment (mikenairn) [inline openshift-ci/README.md]: Probably better not mention individuals here, pinging #test-platform should be enough to get someone's gpg key i would have thought

File: openshift-ci/README.md [hunk 2]
```diff
@@ -33,4 +33,27 @@ After making a change to the base image configuration in your repo, trigger a ne
...
+
+Add their key using `gpg —import <sharedpublickey>.pub`
+
+Download the dockerconfig credentials from quay.io for the account/bot you are using.
+
+Encrypt the file using `gpg --output <outputfile> --encrypt --recipient nmoraiti@redhat.com <dockerconfig>.json
```
Comment (mikenairn) [inline openshift-ci/README.md]: Same here

**Comment by laurafitzgerald** [comment]
Closing in favour of https://github.com/RHCloudServices/integreatly-help/pull/365

**Pull request closed without merging**